vehicle/OVMS.V3/components/vehicle_smarteq/src/vehicle_smarteq.cpp

```diff
@@ -281,6 +294,8 @@ void OvmsVehicleSmartEQ::IncomingFrameCan1(CAN_frame_t* p_frame) {
       break;
     case 0x646:
       mt_use_at_reset->SetValue(CAN_BYTE(1) * 0.1);
+      StandardMetrics.ms_v_charge_kwh_grid->SetValue(mt_use_at_reset->AsFloat());
+      StandardMetrics.ms_v_inv_temp->SetValue(mt_use_at_reset->AsFloat());
```

Comment:
ms_v_inv_temp oder ms_v_charge_kwh_grid was den nun?

Reply:
ms_v_inv_temp, ms_v_charge_kwh_grid ist ein Test ob's doch in die Log's kommt, wäre schöner, aber ist nicht so.